pcdsdevices/spectrometer.py

```diff
@@ -320,6 +320,16 @@ class FZPStates(TwinCATStatePMPS):
     config = UpCpt(state_count=15, motor_count=3)
 
 
+class SOLIDATTStates(TwinCATStatePMPS):
```

Comment:
There are a lot of solid attenuators out there. Could you name this something more specific - a verbose `TMOSpectrometerAttenuatorStates` perhaps?

Reply:
good point. I will do that